src/test/java/org/apache/maven/plugins/dependency/resolvers/ExcludeReactorProjectsDependencyFilterTest.java

```diff
@@ -163,8 +158,6 @@ public Dependency getDependency() {
             }
         };
 
-        final boolean result = filter.accept( node , Collections.<Node>emptyList() );
-
-        assertThat( result, is( true ));
+        assertTrue( filter.accept( node , Collections.<Node>emptyList() ) );
```

Comment:
Here as well and use AssertJ instead of Java Hamcrest.

Reply:
This is standard JUnit, not Hamcrest. AssertJ would add an extra dependency we don't currently use or need.